srtcore/api.cpp

```diff
@@ -656,6 +656,9 @@ int srt::CUDTUnited::newConnection(const SRTSOCKET     listen,
             HLOGC(cnlog.Debug, log << "newConnection: mapping peer " << ns->m_PeerID
                     << " to that socket (" << ns->m_SocketID << ")");
             m_PeerRec[ns->getPeerSpec()].insert(ns->m_SocketID);
+
+            LOGC(cnlog.Note, log << "New connection @" << ns->m_SocketID << " on listener @" << listen
+                << " (" << ns->m_SelfAddr.str() << ") from @" << ns->m_PeerID << " (" << peer.str() << ")");
```

Comment:
I would put the @ socket id first here, that's kinda standard in all logs that use `CONID()` expression (or even use it here, if available - `ns->CONID()` ?).

And additionally if you mention the peer id I think it would be better to mark it explicitly (just to avoid confusion in case of searching or using grep):
`") from peer=@"<< ns->m_PeerID` ...

Reply:
`ns` is of the type `CUDTSocket`, while `CONNID()` if a member function of the `CUDT` class.
It would have to be `ns->m_UDT->CONNID()`, but `m_UDT` is private.

Changing the message to:
```
@121901309 connection created on listener @121901312 (127.0.0.1:4200) from peer @483503516 (127.0.0.1:55596)
```
Without the `=` after `peer=`, cause it doesn't look so nice.